jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/BuildImageMojoIntegrationTest.java

```diff
@@ -535,8 +535,14 @@ public void testExecute_complexProperties()
   }
 
   @Test
-  public void testExecute_skipJibGoal() throws VerificationException, IOException {
-    SkippedGoalVerifier.verifyGoalIsSkipped(skippedTestProject, BuildImageMojo.GOAL_NAME);
+  public void testExecute_jibSkip() throws VerificationException, IOException {
+    SkippedGoalVerifier.verifyJibSkip(skippedTestProject, BuildImageMojo.GOAL_NAME);
+  }
+
+  @Test
+  public void testExecute_jibContainerize() throws VerificationException, IOException {
+    SkippedGoalVerifier.verifyJibContainerize(
+        skippedTestProject, BuildDockerMojo.GOAL_NAME, ":foo");
```

Comment:
How about `:foo` --> `:nonexistingmodule`? It isn't clear if the artifactId of `skippedTestProject` is `foo` or not. Maybe we rename the test for more clarity too: `_jibContainerize_skipUnspecifiedModules` or something along that line. Also, can't we use `simpleTestProject` or `emptyTestProject` instead of `skippedTestProject`? "Skipped" makes it more confusing.

Reply:
Good point:
  - I renamed the `SkippedGoalVerifier.verifyJibContainerize()` &rarr; `verifyJibContainerizeSkips()`, and it no longer takes a module name but instead uses `noGroup:noArtifact` internally.
  - I renamed the test methods to `_jibContainerizeSkips()` and they use either `simpleTestProject` or `emptyTestProject`.